ouroboros-consensus-diffusion/CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 # Changelog entries
 
+<a id='changelog-0.6.0.0'></a>
+## 0.6.0.0 — 2023-05-19
+
+### Patch
+
+- Set ouroboros-network-protocols dependency to ==0.5.0.0
```

Comment:
This was just an intermediary state in #85, `ouroboros-network-protocols` is now back at `^>=0.5`.